docs/data-sources/node.md

```diff
@@ -29,7 +29,7 @@ data "wallarm_node" "aws_staging" {
 ```
 
 ```hcl
-# Looks up for details on WAF nodes with the specific type (can be "cloud_node" or "node")
+# Looks up for details on Wallarm nodes with the specific type (can be "cloud_node" or "node")
```

Comment:
(can be "cloud_node" or "node")? we support only one type

Reply:
cdn node?

Reply:
Example is deleted. In output of data source there still can be the "type" information for the node. The corresponding link is added. CDN node is not supported.